model/variableSheet.h

```diff
@@ -28,7 +28,7 @@ namespace minsky
   class VariableSheet: public ParVarSheet
   {	  
   public:
-    bool variableSelector(ItemPtr i) override {return i->variableCast() && i->variableCast()->type()!=VariableType::parameter;}
+    bool variableSelector(ItemPtr i) override {return i->variableCast() && i->variableCast()->type()!=VariableType::parameter && i->variableCast()->attachedToDefiningVar();}
```

Comment:
I prefer
if (auto v=i->variableCast())
   return v->type()!=VariableType::parameter && v->attachedToDefiningVariable();
return false;
It eliminates 2 unnecessary calls to variableCast()